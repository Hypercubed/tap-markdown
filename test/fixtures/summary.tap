TAP version 13
# t.plan
ok 1 should be equal
# t.end
ok 2 should be equal
ok 3 should be equal

1..3
# tests 3
# pass  3

# ok
