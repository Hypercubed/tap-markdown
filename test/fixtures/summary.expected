# Tests

- âœ” t.plan [pass: 1, fail: 0]
- âœ” t.end [pass: 2, fail: 0]

# Summary

- planned: 3
- assertions: 3
- pass: 3
- fail: 0

